bin/bridge-client/src/args.rs

```diff
@@ -71,6 +71,9 @@ pub(crate) struct Cli {
         description = "retry count for the rocksdb database (default: 3)"
     )]
     pub retry_count: Option<u16>,
+
+    #[argh(option, description = "duty timeout duration in seconds (default: 65)")]
+    pub duty_timeout_duration: Option<u64>,
```

Comment:
Can I annoy you and ask you, since you're editing the `args.rs` and this might be fresh on your mind, to add docstring to all of these fields? The `argh` macros help but they don't provide LSP introspection in the editor/IDE.

Reply:
Done